app/theme.sass

```diff
@@ -45,12 +46,17 @@ $navbar-height: 0
 
 nav.primary
   background-color: $primary
-  padding: 0.25rem 1.5rem
+  padding: 1.5rem 1.5rem
   font-size: 1.2rem
   .brand
-    color: $white
-    font-size: 1.3em
-    padding-right: 0.5rem
+    width: 53px
```

Comment:
I don't really want to change the nav, I want it to be as generic as possible

Reply:
Sure. I made a compromise where if you add `CUSTOM_BRAND_URL` it will render logo.

Reply:
If you still don't like it then I can remove it.